### ISSUE TITLE
how to generate output using generate.py without shuffling?

### ISSUE THREAD

**Issue by kaushal0494** [pytorch/fairseq#2535]
@myleott I am trying to use the mBART generative interface to generated output in the Hindi language (from English). I fine-tuning the model with English-Hindi parallel dataset and results are good.

For my ongoing work, what I want is: "can we generate the Hindi output such that the order of sentences should not be change after generation? (i.e., English sentence order and generated Hindi sentence order should be same, there should not be any shuffling)" How can we achieve the same? Waiting for your response. Thank you!

**Comment by masonreznov** [comment]
You should try `fairseq-interactive`. It genarates the output in the same order.

**Comment by kaushal0494** [comment]
Thanks, @masonreznov
